static/js/compose.js

```diff
@@ -315,7 +315,7 @@ exports.cancel = function () {
 };
 
 exports.empty_topic_placeholder = function () {
-    return empty_topic_placeholder;
+    return i18n.t(empty_topic_placeholder);
```

Comment:
We should leave a comment where empty_topic_placeholder is defined to the effect that i18n is not loaded yet, so we have to translate it here, and references to empty_topic_placeholder within this file should use compose.empty_topic_placeholder rather than calling it directly.